src/components/KDropdownMenu/KDropdownItem.vue

```diff
@@ -158,17 +158,17 @@ const availableComponents = computed((): DropdownItemRenderedRecord => ({
 li.k-dropdown-item {
   align-items: center;
   display: flex;
-  font-size: 16px;
+  font-size: var(--type-md, 16px);
   line-height: 1;
 
   &:not(:first-of-type).has-divider {
-    $k-dropdown-item-divider-container-height: 24px; // set to the same value as --spacing-lg without the units
+    $k-dropdown-item-divider-container-height: var(--spacing-lg, 24px); // set to the same value as --spacing-lg without the units
```

Comment:
Can you use the spacing Sass utils here?
```suggestion
    $k-dropdown-item-divider-container-height: var(--spacing-lg, spacing(lg); // set to the same value as --spacing-lg without the units
```

Same comment for others

Reply:
Done